app/views/api/v1/models/_inbox.json.jbuilder

```diff
@@ -67,7 +69,7 @@ if resource.email?
   ## IMAP
   if Current.account_user&.administrator?
```

Comment:
lets hide the whole blocks that we hide for administrator, not just the password.  ideally we only need to show the inbox attributes and probably the channel type..  or we can simplify this an create a thin model template for inbox which only exposes the necessary attributes